controllers/comments.controller.js

```diff
@@ -0,0 +1,11 @@
+const { removeComment } = require("../models/comments.model")
+
+exports.deleteComment = (req, res, next) => {
+    const { comment_id } = req.params;
+
+    removeComment(comment_id)
+        .then(() => {
+            res.status(204).end();
+        })
```

Comment:
Nice use of end() ... you could make this even more succinct and just send back the status ... take a look [here](http://expressjs.com/en/5x/api.html#res.sendStatus)